class-09-joins-and-relations/lab/zip-codes/scripts/search.js

```diff
@@ -2,6 +2,55 @@
 
   // TODO: Write the code to populate your filters, and enable the search queries here in search.js
   // TODO: You will also interact with the map.js file here
+  function mapFunc(){}
 
+  mapFunc.zipSearch = function(){
+    $('form').on('submit', function(event){
+      event.preventDefault();
+      if($('input[name="zip"]')){
+        initMap.updateZip();
```

Comment:
it might be clearer to do your SQL query here and just pass the result to initMap, especially since you're already checking $('input[name="zip"]') in this method.